gapic-generator-java/src/test/proto/auto_populate_field_testing.proto

```diff
@@ -0,0 +1,144 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/field_behavior.proto";
+import "google/api/field_info.proto";
+import "google/api/resource.proto";
+import "google/longrunning/operations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/rpc/status.proto";
+import "google/protobuf/empty.proto";
+
+package google.auto.populate.field;
+
+option java_package = "com.google.auto.populate.field";
+option java_multiple_files = true;
+
+// This service is meant for testing all scenarios related to
+// the field AutoPopulateFieldTesting feature, including but not limited to the example in gapic-showcase's echo.proto
+service AutoPopulateFieldTesting {
+  option (google.api.default_host) = "localhost:7469";
+
+  // This unary method simply echoes the request. request_id and second_request_id *should* be autopopulated on the request if not set.
+  rpc AutoPopulateFieldTestingEcho(AutoPopulateFieldTestingEchoRequest) returns (AutoPopulateFieldTestingEchoResponse) {
+    option (google.api.http) = {
+      post: "/v1beta1/AutoPopulateFieldTesting:echo"
+      body: "*"
+    };
+  }
+
+  // This server-side streaming method splits the given content into words and will pass each word back
+  // through the stream. This method should not have any autopopulated fields.
+  rpc AutoPopulateFieldTestingExpand(AutoPopulateFieldTestingExpandRequest) returns (stream AutoPopulateFieldTestingEchoResponse) {
+    option (google.api.http) = {
+      post: "/v1beta1/AutoPopulateFieldTesting:expand"
+      body: "*"
+    };
+  }
+
+  // This client-side streaming method will collect the words given to it. When the stream is closed
+  // by the client, this method will return the a concatenation of the strings
+  // passed to it. This method should not have any autopopulated fields.
+  rpc AutoPopulateFieldTestingCollect(stream AutoPopulateFieldTestingEchoRequest) returns (AutoPopulateFieldTestingEchoResponse) {
+    option (google.api.http) = {
+      post: "/v1beta1/AutoPopulateFieldTesting:collect"
+      body: "*"
+    };
+  }
+
+  // This bidirectional streaming method, upon receiving a request on the stream, will pass the same
+  // content back on the stream. This method should not have any autopopulated fields.
+  rpc AutoPopulateFieldTestingChat(stream AutoPopulateFieldTestingEchoRequest) returns (stream AutoPopulateFieldTestingEchoResponse);
+}
```

Comment:
These don't have autopopulated fields b/c they're streaming RPCs, right?

Reply:
Yup, that's correct.

Reply:
Could you add a quick blurb that this is b/c it is a streaming RPC? I saw that it has `AutoPopulateFieldTestingEchoRequest` and forgot about streaming.